src/Components/Endpoints/src/RazorComponentEndpointInvoker.cs

```diff
@@ -69,6 +78,37 @@ private async Task RenderComponentCore()
         await writer.FlushAsync();
     }
 
+    private Task<bool> TryValidateRequestAsync(out bool isPost)
+    {
+        isPost = HttpMethods.IsPost(_context.Request.Method);
+        if (isPost)
+        {
+            return Task.FromResult(TrySetFormHandler());
+        }
+
+        return Task.FromResult(true);
+    }
+
+    private bool TrySetFormHandler()
+    {
+        var handler = _context.Request.Path.Value;
```

Comment:
The "default" handler is the request path, but once we'll support multiple forms, the "handler" query string parameter is what will override the "default" value.

Reply:
The "handler" name is consistent with RazorPages